homeassistant/components/advantage_air/binary_sensor.py

```diff
@@ -1,33 +1,38 @@
 """Binary Sensor platform for Advantage Air integration."""
 
-from homeassistant.components.advantage_air import AdvantageAirEntity
 from homeassistant.components.binary_sensor import (
     DEVICE_CLASS_MOTION,
     DEVICE_CLASS_PROBLEM,
     BinarySensorEntity,
 )
 
 from .const import DOMAIN as ADVANTAGE_AIR_DOMAIN
+from .entity import AdvantageAirEntity
+
+PARALLEL_UPDATES = 0
 
 
 async def async_setup_entry(hass, config_entry, async_add_entities):
     """Set up AdvantageAir motion platform."""
 
     instance = hass.data[ADVANTAGE_AIR_DOMAIN][config_entry.entry_id]
 
-    if "aircons" in instance["coordinator"].data:
-        entities = []
-        for ac_key, ac_device in instance["coordinator"].data["aircons"].items():
-            entities.append(AdvantageAirZoneFilter(instance, ac_key))
-            for zone_key, zone in ac_device["zones"].items():
-                # Only add motion sensor when motion is enabled
-                if zone["motionConfig"] == 2:
-                    entities.append(AdvantageAirZoneMotion(instance, ac_key, zone_key))
-        async_add_entities(entities)
+    entities = []
+    for ac_key, ac_device in instance["coordinator"].data["aircons"].items():
+        entities.append(AdvantageAirZoneFilter(instance, ac_key))
+        for zone_key, zone in ac_device["zones"].items():
+            # Only add motion sensor when motion is enabled
+            if zone["motionConfig"] >= 2:
+                entities.append(AdvantageAirZoneMotion(instance, ac_key, zone_key))
+    async_add_entities(entities)
 
 
 class AdvantageAirZoneFilter(AdvantageAirEntity, BinarySensorEntity):
-    """AdvantageAir Filter."""
+    """Advantage Air Filter."""
+
+    def __init__(self, instance, ac_key):
```

Comment:
Remove this.

Reply:
@MartinHjelmare not sure i follow the intent of this request?